### PR TITLE
WEB-386. added logging when get consumer secret

### PR DESCRIPTION
**Pull request by TamoshaytisV**
We need to catch api error and write it to logs when submitting new license

### CODE REVIEW

**Comment by leha2000** [comment]
please add the license to the error message

**Comment by leha2000** [comment]
:+1:

**Comment by TamoshaytisV** [comment]
@auraz please review

File: lms/djangoapps/labster_course_license/views.py [hunk 1]
```diff
@@ -172,10 +172,11 @@ def set_license(request, course, ccx):
 
     try:
         consumer_key, secret_key = get_consumer_secret(request.user, license)
-    except LabsterApiError:
+    except LabsterApiError as api_err:
         messages.error(
             request, _('There are some issues with applying your license. Please try again in a few minutes.')
         )
+        log.error("Get consumer secret for license {}: {}".format(license, api_err))
```
Comment (auraz) [inline lms/djangoapps/labster_course_license/views.py]: Unable to get?

Comment (TamoshaytisV) [inline lms/djangoapps/labster_course_license/views.py]: here is the code that showsto Nicole an error

```
try:
    consumer_key, secret_key = get_consumer_secret(request.user, license)
except LabsterApiError:
    messages.error(
        request, _('There are some issues with applying your license. Please try again in a few minutes.')
    )
    return redirect(url)
```

here we do not know which API exception is raised

Comment (auraz) [inline lms/djangoapps/labster_course_license/views.py]: I mean, that error text should be more precise: "Unable to get consumer secret for license"

**Comment by TamoshaytisV** [comment]
@avkoval please review asap

**Comment by avkoval** [comment]
:+1:

**Pull request merged**